docs/start/framework/react/quick-start.md

```
@@ -5,82 +5,37 @@ title: Quick Start
-depending on your package manager of choice. You'll be prompted to add things like Tailwind, eslint, and a ton of other options.
+### CLI
 
-You can also clone and run the [Basic](https://github.com/TanStack/router/tree/main/examples/react/start-basic) example right away with the following commands:
+Run:
 
-```bash
-npx gitpick TanStack/router/tree/main/examples/react/start-basic start-basic
-cd start-basic
-npm install
-npm run dev
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)